huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_mysql_instance.go

```diff
@@ -1223,6 +1223,21 @@ func switchSQLFilter(client *golangsdk.ServiceClient, instanceId string, isSQLFi
 	return nil
 }
 
+func checkGaussDBMySQLJobFinish(ctx context.Context, client *golangsdk.ServiceClient, jobID string, timeout time.Duration) error {
```

Comment:
this func is not used in this resource, how about put it in the restore resource?

Reply:
it will be reused when retry is added to update methods